app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    _, src, dst = command.split()
```

Comment:
The function should verify that the command string starts with 'cp' and contains exactly three elements separated by spaces. This is a requirement according to the task checklist: 'Make sure you check that the input string contains 3 elements separated by space, and has `cp` on a first place.'